src/iop/highlights.c

```diff
@@ -24,6 +24,8 @@
 #include <string.h>
 #include "bauhaus/bauhaus.h"
 #include "common/opencl.h"
+#include "common/imagebuf.h"
```

Comment:
both line are superfluous at this point.

Reply:
right again. Forgot to remove after compile-testing.